ddtrace/contrib/django/patch.py

```diff
@@ -348,6 +349,10 @@ def traced_get_response(django, pin, func, instance, args, kwargs):
 @trace_utils.with_traced_module
 def traced_template_render(django, pin, wrapped, instance, args, kwargs):
     """Instrument django.template.base.Template.render for tracing template rendering."""
+    # DEV: Check here in case this setting is configured after a template has been instrumented
+    if not config.django.instrument_templates:
```

Comment:
wonder if we should do this for the middleware and database patches too...

Reply:
probably, that was the thought I had as well when writing it. I know some people have had trouble with disabling properly.